### ISSUE TITLE
grad is None

### ISSUE THREAD

**Issue by LijunZhang01** [YuxiXie/V-DPO#2]
Excellent work, but the model in your public code cannot be trained. During training, the gradient is None and the loss does not change. What is the reason?

**Comment by Scipio1996** [comment]
> Excellent work, but the model in your public code cannot be trained. During training, the gradient is None and the loss does not change. What is the reason?

Hi， I also find the loss does not change, have you solved this problem?

**Comment by LijunZhang01** [comment]
> > Excellent work, but the model in your public code cannot be trained. During training, the gradient is None and the loss does not change. What is the reason?
> 
> Hi， I also find the loss does not change, have you solved this problem?

Increase the learning rate（0.1）.But strangely, the learning rate in the paper is 1e-6
